### PR TITLE
Adds tabs plugin

### PR DESCRIPTION
**Pull request by FinalAngel**
- Adds tabs plugin
- Modifies carousel to use different template names

### CODE REVIEW

File: djangocms_bootstrap4/contrib/bootstrap4_tabs/constants.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ('nav-fill', _('Fill')),
+    ('nav-justified', _('Justified')),
+    ('justify-content-start', _('Justify start')),
+    ('justify-content-center', _('Justify center')),
+    ('justify-content-end', _('Justify end')),
+    ('flex-column', _('Column')),
+)
+
+TAB_EFFECT_CHOICES = (
+    ('fade', _('Fade')),
```
Comment (vxsx) [inline djangocms_bootstrap4/contrib/bootstrap4_tabs/constants.py:23]: would be awesome to have all of this actually configurable

**Review by vxsx — APPROVED** [review]
looks ok to me (i know nothing of backend)
would love to have options configurable (since in my experience the default options are never used much), but can live without it for now

File: djangocms_bootstrap4/contrib/bootstrap4_tabs/constants.py (hunk 1 above)
Comment (FinalAngel) [reply djangocms_bootstrap4/contrib/bootstrap4_tabs/constants.py:23]: Will think about this once I'm doing cleanup :)

**Comment by filwaitman** [comment]
:+1:

**Pull request merged**